website/src/4.x/docs/guides/remote-assets.md

```diff
@@ -101,3 +101,30 @@ new Repack.RepackPlugin({
 ```
 
 The final step is to upload your remote assets to your CDN, which is located at `publicPath`, and then host them from that location, which will make them available to users of your app.
+
+### Customizing Asset Path
+
+The `assetPath` option offers finer control over how remote asset paths are constructed. This feature allows you to define a custom function for modifying paths, which can be helpful if you need to apply custom naming conventions or add extra directory layers.
+Specified pattern will be applied to both the generated folder path and URL. If `assetPath` is not provided, the default pattern will be used.
```

Comment:
let's specify the default pattern explicitly here

Reply:
I got an idea, since we already have the description for default behaviour, maybe we could just link it? Check out the latest commit

Reply:
yeah, that works 🎉